mem_sql/pom.xml

```diff
@@ -74,6 +94,18 @@
             <artifactId>pipeline_api</artifactId>
             <version>2.7.0.1808-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hbase</groupId>
+            <artifactId>hbase-common</artifactId>
+            <version>1.0.0-cdh5.6.0</version>
```

Comment:
has this library version not had an upgrade in 1.5 years?

Reply:
It has not.  Feel free to fix if you would like.